docs/src/apimanual.md

```diff
@@ -1325,11 +1325,18 @@ end
 
 const BRIDGED = MOI.instantiate(OPTIMIZER_CONSTRUCTOR, with_bridge_type = Float64)
 const CONFIG = MOIT.TestConfig(atol=1e-6, rtol=1e-6)
-
+```
+`MOI.Test.unittests` is a dictionary having "test name":test function as a key:value pair and `MOIT.unittest(model, config)` runs all of these tests. You can use `keys(MOI.Test.unittests)` to obtain a list of keys. To exclude any particular tests you can pass vector of there keys to the `exclude` keyword argument of unittest.
```

Comment:
This applies, not just for `unittests`, but all of the `MOI.Test.XXX` functions.

You want something like

Test functions like `MOI.Test.unittest` and `MOI.Test.modificationtest` are wrappers around corresponding dictionaries `MOI.Test.unittests` and `MOI.Test.modificationtests`. The keys of each dictionary (strings describing the test) map to functions that take two arguments: an optimizer and a `MOI.Test.TestConfig` object. Exclude tests by passing a vector of strings corresponding to the test keys you want to exclude as the third positional argument to the test function (e.g., `MOI.Test.unittest`). Print a list of all keys using `println.(keys(MOI.Test.unittests))`.

You could also put this paragraph below the code, so that someone can copy-paste the code block in one piece.